### ISSUE TITLE
Laravel 5.5.39 breaks making a migration?

### ISSUE THREAD

**Issue by CrixuAMG** [laravel/framework#23503]
When running `php artisan make:migration table_name`, the following exception is thrown:
```php
 include(/home/vagrant/dev/project/vendor/composer/../../database/migrations/2018_03_12_150225_create_table_name_table.php): failed to open stream: No such file or directory
```

The file is indeed not created when viewing the directory. In 5.5.38 it works perfectly.

PHP version: 7.1.10-1+ubuntu16.04.1+deb.sury.org+1 (cli) (built: Sep 29 2017 17:04:25) ( NTS )

**Comment by browner12** [comment]
could you run your migration with `-vvv` to hopefully provide some more debug information?

**Comment by CrixuAMG** [comment]
Hmmm... I've upgraded and downgraded from 5.5.38 to 5.5.39 a couple of times, but I cannot reproduce the issue.
Now I'm wondering if it was somehow a caching issue, but I also ran `cache:clear`.

**Comment by edgarsn** [comment]
`composer dump-autoload` didn't help?

**Comment by CrixuAMG** [comment]
@edgarsn I've tried that as well, it did not have any effect, the same goes for `config:clear`.

**Comment by CrixuAMG** [comment]
The issue occurred again, it seems like when the command is repeated too often it locks down, but only for migrations?
I can't repeatedly reproduce the issue, which makes me think that the OS could be a factor in this.
Using the latest version of Laravel Homestead to locally develop.
In Homestead I also use the commands, after cd'ing to the root of the Laravel project.

**Comment by browner12** [comment]
not that this will necessarily fix the problem, but we did release a new version of Homestead yesterday you could try.

**Comment by CrixuAMG** [comment]
I tried upgrading (from v7.2.0 to v7.3.0), which went fine.
When running `vagrant box update` I get the following message:
```
==> homestead-7: Box 'laravel/homestead' not installed, can't check for updates.
```
while in the ~/Homestead directory.
When running `vagrant box list` I can see that homestead is actually installed.

About the current issue with migrations, I've tried creating 15 different migrations, all went fine so far.
I'll keep on testing today and tomorrow and hope it's fixed!
